src/crawler/crawler.service.ts

```
@@ -1,150 +1,360 @@
+            salary,
+            tags,
+            postedAt: "not specified",
+            applyUrl,
+            source: "Remote.co",
+            description,
+            type: "Remote",
+          });
+        } catch (jobErr) {
+          this.logger.warn(`Failed to scrape one Remote.co job: ${jobErr.message}`);
+          continue;
+        }
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)